Makefile

```diff
@@ -271,7 +271,22 @@ verify-modules:
         exit 1 ; \
     fi
 
+golint:
+	@if [ $$(golint \
+            $$(ls -d */ | grep -v "vendor/" | sed -e "s/^\(.*\)$$/\1.../g") | \
+        grep -v "generated.deepcopy.go:" | \
+        wc -l) -eq 0 ] ; then \
```

Comment:
why not use go list ./... here?

Reply:
I think I changed this - want to re-check?